app/views/layouts/application.html.erb

```diff
@@ -6,14 +6,23 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "application", media: all, "data-turbo-track": "reload" %>
```

Comment:
- [ ] @Team Great work implementing the changes requested, however am getting the error below when I run your app, this is because all in the stylesheet_link_tag is supposed to be a string, not a variable.
![image](https://github.com/jcmunav63/my-ror-blog/assets/61408860/4868ea02-9354-4f73-bbca-6e79c9c54e0d)